src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRail.tsx

```diff
@@ -49,14 +49,19 @@ export const AuctionArtworksRail: React.FC<AuctionArtworksRailProps> = ({
             </Text>
           )}
         </Box>
-
-        <Box height={AUCTION_ARTWORKS_RAIL_HEIGHT}>
-          {isEnteredView ? (
-            <AuctionArtworksRailArtworksQueryRenderer id={sale.internalID} />
-          ) : (
-            <AuctionArtworksRailPlaceholder />
-          )}
-        </Box>
+        {/* Only showing Artwork Images for Current and Past Auctions */}
+        {(!sale.isLiveOpen && sale.status == "open") ||
+        (!sale.isLiveOpen && sale.status == "closed") ? (
+          <Box height={AUCTION_ARTWORKS_RAIL_HEIGHT}>
+            {isEnteredView ? (
+              <AuctionArtworksRailArtworksQueryRenderer id={sale.internalID} />
+            ) : (
+              <AuctionArtworksRailPlaceholder />
+            )}
+          </Box>
+        ) : (
+          <></>
+        )}
```

Comment:
We could simplify this logic a bit by collapsing some of the args and removing the ternary:
```tsx
{!sale.isLiveOpen && (sale.status === "open" || sale.status === "closed) && (
  <Box>...</Box>
)}
```

Reply:
Or better yet, move the logic into a self-documenting variable name:

```
const isCurrentOrPastAuction = !sale.isLiveOpen && (sale.status === "open" || sale.status === "closed)

...

{isCurrentOrPastAuction && (
  <Box>...</Box>
)}

Reply:
Removed this also.